testing/test_recwarn.py

```diff
@@ -217,17 +217,13 @@ def test_as_contextmanager(self):
         excinfo.match(re.escape(message_template.format(warning_classes,
                                                         [each.message for each in warninfo])))
 
-
     def test_record(self):
         with pytest.warns(UserWarning) as record:
             warnings.warn("user", UserWarning)
 
         assert len(record) == 1
         assert str(record[0].message) == "user"
 
-        print(repr(record[0]))
```

Comment:
This was testing that `record` was a `RecordedWarning`, which has been replaced by `warnings` own tuple.